docs/pywbemcli/commands.rst

```diff
@@ -1559,6 +1563,77 @@ Example:
 See :ref:`pywbemcli server remove-mof --help` for the exact help output of the
 command.
 
+.. _`Server schema command`:
+
+``server schema`` command
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``server schema`` command returns information on the  :ref:`Schema`
+defined for classes in the namespace(s) of the :term:`current connection`
```

Comment:
The phrase "Schema defined for classes" does not describe it well. This focuses on the prefix in the class names, but the schema is more than just the prefix.

I suggest saying "... on the :ref:`schema`s in the namespaces of ..."

Reply:
DONE